### DISCUSSION TITLE
Need help with photo attachments in iOS environment

### DISCUSSION THREAD

**Discussion by Ysidros** [opengisch/QField#4238] [Q&A]
Hello there,

I am trying to add a photo field to a layer and I did setup everything as described in the documentation for the attachment widget. The field works as far as it does open the photo app on the iPad or lets me choose a photo from the camera roll. The photos are saved within the project folder of QField. So far, so good.
The only problem is, that no data is entered into the attribute table, the field remains empty. 
Is there a way to add the filename of the photo or any other info into the attribute table?

cheers,
Ysidros

**Comment by VxTedxV** [comment]
Be careful with the length of the photo path according to the field config ? (ex : limited to 254 characters)

**Comment by VxTedxV** [comment]
You're welcome !
Open another issue if you don't find any answer about this geotag issue.

**Reply by Ysidros** [reply]
Sure, I will.
